WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -261,10 +256,24 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     /// Presents the Login Epilogue, in the specified NavigationController.
     ///
     func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, onDismiss: @escaping () -> Void) {
```

Comment:
Perhaps we could consider adding unit tests to `AuthenticationManagerTests` to test this method.

Reply:
Good point! I added some tests here: 36aac5be71784b5d29d22e2d483ab2128614b872.